.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    # services:
+    #         mongodb:
+    #             image: mongo:4.2.3
+    #             command: --setParameter ttlMonitorSleepSecs=2
+    #             ports:
+    #               - 27017:27017
```

Comment:
Please remove comments

Reply:
I have leave them there because when that would be supported we can use the service. Would you like me to remove them anyway?

Reply:
Yes please, thanks